tests/calculations/test_singlepoint.py

```diff
@@ -107,6 +110,24 @@ def test_sp_nostruct(fixture_sandbox, generate_calc_job, model_folder, janus_cod
         generate_calc_job(fixture_sandbox, entry_point_name, inputs)
 
 
+def test_sp_nomodel(fixture_sandbox, generate_calc_job, config_folder, janus_code):
+    """Test singlepoint calculation with error input"""
+    entry_point_name = "mlip.sp"
+
+    nacl = bulk("NaCl", "rocksalt", a=5.63)
+    write("NaCl.cif", nacl)
```

Comment:
Given that this isn't testing the config reader, I'd consider defining struct (and writing here) using temporary files, which would mean you don't need to unlink.

Reply:
To do this you can either use `tempfile.NamedTemporaryFile` or use the `tmp_path` fixture from pytest (pass `tmp_path` as an arugment) and put the written file in `tmp_path / "NaCl.cif"`

Reply:
I just changed it in a way that I don't actually need a file